test/e2e/e2e_test.go

```diff
@@ -27,3 +30,16 @@ func TestE2E(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "e2e tests")
 }
+
+var _ = BeforeSuite(func() {
+	logrus.SetLevel(logrus.DebugLevel)
+	logrus.SetFormatter(&logrus.TextFormatter{FullTimestamp: true})
+	logrus.SetOutput(GinkgoWriter)
+	log := logrus.NewEntry(logrus.StandardLogger())
+	Expect(log).NotTo(BeNil())
+
+	conf, err := realfakerp.NewConfig(log, true)
+	Expect(err).NotTo(HaveOccurred())
+	Expect(conf).NotTo(BeNil())
+	realfakerp.StartServer(log, conf, realfakerp.LocalHttpAddr)
```

Comment:
We should probably not try to run the fake RP here since it is going to run in production tests too?

Reply:
i'll move this into the `NewClientFromEnvironment` azure client at test/clients/azure/client.go